### PR TITLE
Ruby sample code

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: sample-code/ruby/spec/android_basic_interactions_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  appium_lib: {
+    sauce_username:   ENV['SAUCE_LABS'] ? ENV['SAUCE_USERNAME'] : nil,
+    sauce_access_key: ENV['SAUCE_LABS'] ? ENV['SAUCE_ACCESS_KEY'] : nil,
+    wait: 60
+  }
+}
+
+describe 'Basic Android interactions' do
+
+  @driver
```
Comment (KazuCocoa) [inline sample-code/ruby/spec/android_basic_interactions_spec.rb]: You can remove this line

File: sample-code/ruby/spec/android_create_web_session_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    wait: 60
+  }
+}
+
+
+describe 'Create Chrome web session' do
+  it 'should create and destroy Android browser session' do
+    @driver = Appium::Driver.new(desired_caps, true).start_driver
+
+    @driver.get('https://www.google.com');
```
Comment (KazuCocoa) [inline sample-code/ruby/spec/android_create_web_session_spec.rb]: You can remove `;` at the end of lines

File: sample-code/ruby/spec/android_selectors_spec.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  appium_lib: {
+    sauce_username:   ENV['SAUCE_LABS'] ? ENV['SAUCE_USERNAME'] : nil,
+    sauce_access_key: ENV['SAUCE_LABS'] ? ENV['SAUCE_ACCESS_KEY'] : nil,
+    wait: 60
+  }
+}
+
+describe 'Basic Android selectors' do
+
+  @driver
```
Comment (KazuCocoa) [inline sample-code/ruby/spec/android_selectors_spec.rb]: You can remove this line.

File: sample-code/ruby/Gemfile [hunk 4]
```diff
@@ -0,0 +1,10 @@
+source 'https://www.rubygems.org'
+
+gem 'appium_lib',         '~> 9.7.4'
```
Comment (KazuCocoa) [inline sample-code/ruby/Gemfile]: The latest version is `9.10`. Could you set the line as `'~> 9.10'`?
It means we can download the latest `9.x` version.

**Review by KazuCocoa — COMMENTED** [review]
I left some comments, but I can improve them after merging this PR.

**Comment by dpgraham** [comment]
Fixes pushed

**Review by KazuCocoa — APPROVED** [review]
thanks 👍

**Pull request merged**